composer.json

```diff
@@ -9,18 +9,19 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^8.3",
```

Comment:
Hm, IMO it might be too aggressive for this bundle, no? Also, PHP 8.1 & 8.2 still support security fixes. Even Sf 7.3 requires 8.2

Reply:
If it's a well-grounded change - I'm fine with it. I mean, why exactly should we use 8.3 and makes problems for people who are still on 8.2? Do we use any new syntax from 8.3 that requires as for this move? Of not - I would prefer to keep 8.2 users covered still. But also, what about LTS Symfony 6.4? I suppose we do want to support that version as well, and that version requires >=8.1. Unless we have strong arguments why it should be 8.3 - I think we can make PHP constraints not that strict IMO

Reply:
But ok, we can think about the exact PHP version constraint later before the release, probably it would be more obvious base on the actual changes